doc/s9s.dbversions.1

```diff
@@ -87,7 +87,23 @@ The list will contain the available 3-digit versions separated by a newline char
 .SS Arguments related to list versions operations
 The list operations: 'getDbVersions' (--list-versions) and 'getDb3dVersions' (--list-3d-versions)
```

Comment:
`--list-3d-versions` or `--list-3d`?

Reply:
--list-3d

Reply:
Then please change. Also please check for the in the `--help` option. I saw `--list-3d-versions` there too. Thank you.